session/session.go

```diff
@@ -2810,7 +2811,52 @@ func loadCollationParameter(se *session) (bool, error) {
 	return false, nil
 }
 
-var errResultIsEmpty = dbterror.ClassExecutor.NewStd(errno.ErrResultIsEmpty)
+var (
+	errResultIsEmpty = dbterror.ClassExecutor.NewStd(errno.ErrResultIsEmpty)
+	// DDLJobTables is a list of ddl tables definition.
+	DDLJobTables = []string{
+		fmt.Sprintf("create table %s(job_id bigint not null, reorg int, schema_id bigint, table_id bigint, job_meta longblob, processing bigint, is_drop_schema int, primary key(job_id))", ddl.JobTable),
+		fmt.Sprintf("create table %s(job_id bigint not null, ele_id bigint, curr_ele_id bigint, curr_ele_type blob, start_key blob, end_key blob, physical_id bigint)", ddl.ReorgTable),
```

Comment:
Could you add one  blob column, named metas. I will used it for lightning related information